<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" idbasedtr="true">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Clients</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>384</width>
             <height>499</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QListWidget" name="client_list"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>384</width>
             <height>259</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Info</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QScrollArea" name="scrollArea_4">
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents_4">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>364</width>
                     <height>211</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <layout class="QGridLayout" name="gridLayout_2">
                      <item row="0" column="1">
                       <widget class="QLineEdit" name="lineEdit">
                        <property name="enabled">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                      <item row="2" column="1">
                       <widget class="QListWidget" name="listWidget">
                        <property name="enabled">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="1">
                       <widget class="QLineEdit" name="lineEdit_2">
                        <property name="enabled">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_2">
                        <property name="text">
                         <string>Addres</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="0">
                       <widget class="QLabel" name="label_4">
                        <property name="text">
                         <string>Login</string>
                        </property>
                       </widget>
                      </item>
                      <item row="2" column="0">
                       <widget class="QLabel" name="label_5">
                        <property name="text">
                         <string>Group</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>384</width>
             <height>258</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTextBrowser" name="text_log">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="markdown">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="action_run_server"/>
    <addaction name="action_config_server"/>
    <addaction name="action_stop_server"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_author"/>
   </widget>
   <widget class="QMenu" name="menu_debug_gui">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>_debug_gui</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="action_add_anon"/>
    <addaction name="separator"/>
    <addaction name="action_logout_client"/>
    <addaction name="action_login_client"/>
    <addaction name="action_auth_client"/>
    <addaction name="action_add_group"/>
    <addaction name="action_leave_group"/>
    <addaction name="action_kick_client"/>
    <addaction name="action_send_msg_to_user"/>
    <addaction name="action_send_msg_to_group"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAction_2">
    <property name="title">
     <string>Client</string>
    </property>
    <addaction name="actionlogout"/>
    <addaction name="actionkick"/>
    <addaction name="actionsend_masg"/>
   </widget>
   <addaction name="menuAction"/>
   <addaction name="menuAction_2"/>
   <addaction name="menuAbout"/>
   <addaction name="menu_debug_gui"/>
  </widget>
  <action name="action_run_server">
   <property name="text">
    <string>Run Server</string>
   </property>
  </action>
  <action name="action_stop_server">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop Server</string>
   </property>
  </action>
  <action name="actionServer_Config">
   <property name="text">
    <string>Server Config</string>
   </property>
  </action>
  <action name="action_author">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_config_server">
   <property name="text">
    <string>Config Server</string>
   </property>
  </action>
  <action name="actionKick_Client">
   <property name="text">
    <string>Kick Client</string>
   </property>
  </action>
  <action name="action_add_anon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>connect client</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_logout_client">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>logout client</string>
   </property>
  </action>
  <action name="actionadd_client">
   <property name="text">
    <string>add client</string>
   </property>
  </action>
  <action name="action_auth_client">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>auth client</string>
   </property>
  </action>
  <action name="action_add_group">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>add client to group</string>
   </property>
  </action>
  <action name="action_leave_group">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>leave client from group</string>
   </property>
  </action>
  <action name="action_kick_client">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>kick client</string>
   </property>
  </action>
  <action name="action_debug">
   <property name="text">
    <string>_debug</string>
   </property>
  </action>
  <action name="action_send_msg_to_user">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>send_msg_to_user</string>
   </property>
  </action>
  <action name="action_send_msg_to_group">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>send_msg_to_group</string>
   </property>
  </action>
  <action name="actionkick">
   <property name="text">
    <string>kick</string>
   </property>
  </action>
  <action name="actionlogout">
   <property name="text">
    <string>logout</string>
   </property>
  </action>
  <action name="actionsend_masg">
   <property name="text">
    <string>send message</string>
   </property>
  </action>
  <action name="action_login_client">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>login client</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
